Assets/Scripts/Menu/Page.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Events;
+
+[RequireComponent(typeof(Animator))]
+public class Page : MonoBehaviour
+{
+    [SerializeField] private Button _backButton;
+
+    private Animator _animator;
+
+    public UnityAction BackPressed;
+
+    public void Open()
+    {
+        _animator.SetTrigger("open");
+    }
+
+    public void Close()
+    {
+        _animator.SetTrigger("close");
```

Comment:
https://trello.com/c/UAeMGORw/17-%D0%BE%D1%82%D0%BA%D0%B0%D0%B7-%D0%BE%D1%82-%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BE%D0%B2%D1%8B%D1%85-%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B5%D0%BB%D0%BE%D0%B2